src/Serilog.Sinks.Splunk/Sinks/Splunk/EventCollectorRequest.cs

```diff
@@ -32,6 +32,11 @@ internal SplunkEvent(string logEvent, string source, string sourceType, string h
                 jsonPayLoad = jsonPayLoad + @",""index"":""" + index + @"""";
```

Comment:
Good point on Newtonsoft, my main point for linking to the code was showing how we do it. We basically have an in-memory queue where events are written to which allows us to process things very efficiently in an async manner.